defaults/main.yml

```diff
@@ -99,6 +99,7 @@ cartridge_defaults: {}
 config: null
 cartridge_runtime_params: null
 zone: null
+cartridge_extra_env: null
```

Comment:
`cartridge_instance_env`?

Reply:
You can use this variable for all instances